Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/18F/identity-hostdata.git
-  revision: 56acfd79af4f7290305fefca86230c5c93012f54
-  branch: mitchellhenke/allow-skipping-s3-config
+  revision: e0ada38c45a43e1251dc6f369e547f3525d549a8
+  tag: v3.4.3
   specs:
-    identity-hostdata (3.4.1)
+    identity-hostdata (3.4.2)
```

Comment:
the tag is `v3.4.3` but this says `3.4.2` ... which one is right?

Reply:
Hm, it looks like 3.4.2 was tagged but the version stayed at 3.4.1, so I might just force tag over 3.4.2?